source/synthDrivers/sapi5.py

```diff
@@ -37,9 +37,9 @@ class FunctionHooker(object):
 	def __init__(self,targetDll,importDll,funcName,newFunction):
 		hook=NVDAHelper.localLib.dllImportTableHooks_hookSingle(targetDll,importDll,funcName,newFunction)
 		if hook:
-			print "hooked %s"%funcName
+			print("hooked %s"%funcName)

```

Comment:
Actually, the print statements in this file should have always been log.debug calls. Currently they are just noise going to stdout.